hugo/data/home/home.yml

```diff
@@ -19,10 +19,8 @@ slack:
     server_error: Error! submit again.
     fetch_error: Server error
 
-roadmap:
-  - Ad id nulla consectetur occaecat duis est pariatur. Sunt anim id cillum aliqua enim id ea sit sunt voluptate dolor labore mollit. Excepteur nulla qui minim voluptate ea. Eu ullamco consectetur voluptate amet commodo mollit esse cillum ullamco.
-  - Ad id nulla consectetur occaecat duis est pariatur. Sunt anim id cillum aliqua enim id ea sit sunt voluptate dolor labore mollit. Excepteur nulla qui minim voluptate ea. Eu ullamco consectetur voluptate amet commodo mollit esse cillum ullamco.
-
+ceo_picture: https://avatars2.githubusercontent.com/u/1247608?v=3
```

Comment:
I'd put it under the `roadmap` key, because it has no sense out of that scope; that way if the whole section is dropped, that key will not be dangling.